splunk_eventgen/lib/plugins/generator/jinja.py

```diff
@@ -186,20 +191,34 @@ def gen(self, count, earliest, latest, samplename=None):
                 if self._sample.jinja_count_type in ["line_count", "cycles", "perDayVolume"]:
                     self.jinja_count_type = self._sample.jinja_count_type
             startTime = datetime.datetime.now()
-            working_dir, working_config_file = os.path.split(self.config.configfile)
+
+            # if eventgen is running as Splunk app the configfile is None
+            if self.config.configfile:
+                working_dir, working_config_file = os.path.split(self.config.configfile)
+            else:
+                splunk_home = os.environ["SPLUNK_HOME"]
```

Comment:
Can we provide a default SPLUNK_HOME in case where environment var "SPLUNK_HOME" doesn't exist?

Reply:
When running as modular input we can always get SPLUNK_HOME. Providing a default value may not apply to both Linux and Windows.